types/lru.go

```diff
@@ -0,0 +1,120 @@
+package types
+
+import (
+	"github.com/hashicorp/golang-lru/simplelru"
+)
+
+var CacheStore *Cache = NewCache(20)
```

Comment:
Told ed to delete this, just incase you should delete it to

Reply:
removed!